docs/admin.md

```diff
@@ -0,0 +1,70 @@
+# Admin
+
+## Connect to your cluster
+
+Ensure you are connected to your cluster because Warnet will use your current configuration to generate configurations for your users.
+
+```shell
+$ warnet status
+```
+
+Observe that the output of the command matches your cluster.
+
+## Create an *admin* directory
+
+```shell
+$ mkdir admin
+$ cd admin
+$ warnet admin init
+```
+
+Observe that there are now two folders within the *admin* directory: *namespaces* and *networks*
+
+## The *namespaces* directory
+This directory contains a Helm chart named *two_namespaces_two_users*.
+
+Modify this chart based on the number of teams and users you have.
+
+Deploy the *two_namespaces_two_users* chart.
+
+```shell
+$ cd namespaces
+$ warnet deploy two_namespaces_two_users
```

Comment:
Perhaps:

```suggestion
$ warnet deploy namespace/two_namespaces_two_users
```

Reply:
Thank you. Fixed: https://github.com/bitcoin-dev-project/warnet/pull/616/commits/4460f20a6e927fcb9e49bbf573f15dafd5ed50e4 "admin.md: update namespace deploy command"